src/types/app.ts

```diff
@@ -263,3 +263,4 @@ export type Permission =
   | 'VOUCHER_PLAN_READ'
   | 'VOUCHER_PLAN_WRITE'
   | 'VOUCHER_CODE_EXPORT'
+  | 'null'
```

Comment:
what is this null string type for?

Reply:
`lodestar-app-admin/src/components/project/ProjectCollectionTabs.tsx:21, 27` need the permission type as key, but only string and symbol data type can be the key

Reply:
should modify ProjectCollectionTabs instead of adjusting here
maybe not a good idea to use 'null' string type
But it's alright , modify in the future ~~